Add celery worker prototype

kandski opened the pull request:
add basic celery configuration

kandski committed:
Add celery worker prototype

add basic celery configuration

kandski commented on Dockerfile:
```diff
@@ -2,6 +2,8 @@ FROM python:3.6-alpine
 
 COPY requirements.txt app/
 
+RUN apk --no-cache add gcc musl-dev
```
@xmalmed in fact I'm not sure if it should or shouldn't be here, can you please check that? I think that it was your contribution, and maybe I forgot to check and rebase properly.

xmalmed replied:
Yes, it's mine. And I will introduce it in #63

kandski commented on src/bot_worker/celery.py:
```diff
@@ -0,0 +1,21 @@
+                    worker_concurrency=6,
+                    imports=['bot_worker.tasks', 'iwant_bot.pool'],
+                  )
+worker.conf.beat_schedule = {
```
it is possible to add some task to be scheduled, clean up of database on 1st day of every month for example.

kandski commented on src/bot_worker/tasks.py:
```diff
@@ -0,0 +1,20 @@
+import time
+
+
+# this is a working off-topic example of celery task
```
Just for testing and demonstration purposes

matejak committed:
Implement results storage and SQLAlchemy backend. (#60)

* Add results storage functionality.

* Implement SQLAlchemy backend for requests storage.

* Prepare for Postgres storage backend support.

* Refactor request removal to be more understandable.

* Make the basic storage tests pass.

* Improve naming and add style fixes.

* Fix unit tests.

* Add proper hashing support to activity requests.

* Add the activity request data structure.

It supports deadeline and activity start/end.

* Fix PEP8 violations.

* Fix test with the new request API.

* Change baseimage to python-slim.

This makes installation of Python extensions easy, because
debian-jessie falls into "manylinux" cathegory in which wheels work.

* Add support for postgres dialect of sqlalchemy.

* Skip postgres test if the testing environment is not correct.

* Remove unused import.

kandski committed:
Fix code to comply to PEP8

matejak committed:
Merge branch 'master' into celery

kandski committed:
Fix arguments passing in pool test

kandski committed:
Fix code to comply with PEP8

matejak committed:
Merge branch 'master' into celery

matejak committed:
Fix errors introduced during merge.

matejak committed:
Fix PEP8 errors introduced during merge.
